src/routes.tsx

```diff
@@ -0,0 +1,22 @@
+import { LoginPage } from './AdminPanel/modules/LoginPage';
+import App from './App';
+import { createBrowserRouter } from 'react-router-dom';
+
+const routes = [
+  {
+    path: '/admin-panel/login',
+    element: <LoginPage />,
+  },
+  {
+    path: '/',
+    element: <App />,
+  },
+  {
+    path: '*',
+    element: <div data-cy="error-div">Error: wrong path: {window.location.pathname}</div>,
+  },
+];
+
+const router = createBrowserRouter(routes);
+
+export default router;
```

Comment:
I think it would be better do something like this:
```typescript
import { createBrowserRouter, RouteObject } from 'react-router-dom';

const routes: RouteObject[] = [
  {
    path: '/admin-panel/login',
    element: <LoginPage />,
  },
  {
    path: '/',
    element: <App />,
  },
  {
    path: '*',
    element: <div data-cy="error-div">Error: wrong path: {window.location.pathname}</div>,
  },
];

export default createBrowserRouter(routes);

```

Reply:
resolved